### PR TITLE
Hono metrics stored in InfluxDB and viewable in sample Grafana dashboard

### PR DESCRIPTION
**Pull request by pellmann**
Grafana and InfluxDB docker images in example for metrics. There is an InfluxDB, which is used to store the Hono metrics and a Grafana with a first sample dashboard.

### CODE REVIEW

File: example/pom.xml [hunk 1]
```diff
@@ -78,6 +78,10 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
         <targetPath>${project.build.directory}/hono</targetPath>
       </resource>
       <resource>
+        <directory>${project.basedir}/grafana</directory>
```
Comment (sophokles73) [inline example/pom.xml]: what is this good for?

File: bom/pom.xml [hunk 2]
```diff
@@ -114,6 +116,11 @@
         <version>${dropwizard-metrics.version}</version>
       </dependency>
       <dependency>
+        <groupId>io.dropwizard</groupId>
```
Comment (sophokles73) [inline bom/pom.xml]: should probably be `io.dropwizard.metrics`

File: bom/pom.xml [hunk 3]
```diff
@@ -114,6 +116,11 @@
...
+        <artifactId>dropwizard-metrics-graphite</artifactId>
```
Comment (sophokles73) [inline bom/pom.xml]: should probably be `metrics-graphite`

File: bom/pom.xml [hunk 4]
```diff
@@ -34,6 +34,7 @@
     <assertj-core.version>3.2.0</assertj-core.version>
     <jackson.version>2.7.4</jackson.version>
     <dropwizard-metrics.version>3.2.2</dropwizard-metrics.version>
+    <dropwizard-metrics-graphite.version>1.1.0</dropwizard-metrics-graphite.version>
```
Comment (sophokles73) [inline bom/pom.xml]: should be removed

File: bom/pom.xml [hunk 5]
```diff
@@ -114,6 +116,11 @@
         <version>${dropwizard-metrics.version}</version>
       </dependency>
       <dependency>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-metrics-graphite</artifactId>
+        <version>${dropwizard-metrics-graphite.version}</version>
```
Comment (sophokles73) [inline bom/pom.xml]: should probably be `${dropwizard-metrics.version}`

Comment (pellmann) [inline bom/pom.xml]: I changed now to the aligned version of graphite - so all your proposed changes are made.

File: application/pom.xml [hunk 6]
```diff
@@ -40,6 +40,10 @@
       <groupId>io.dropwizard.metrics</groupId>
       <artifactId>metrics-jvm</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard</groupId>
```
Comment (sophokles73) [inline application/pom.xml]: `io.dropwizard.metrics`

File: application/pom.xml [hunk 7]
```diff
@@ -40,6 +40,10 @@
...
+      <artifactId>dropwizard-metrics-graphite</artifactId>
```
Comment (sophokles73) [inline application/pom.xml]: `metrics-graphite`

Comment (pellmann) [inline application/pom.xml]: Ups - changed.

File: site/content/getting-started.md [hunk 8]
```diff
@@ -183,3 +183,7 @@ In order to start up the instance again:
 ~~~sh
 ~/hono/example/target/hono$ docker stack deploy -c docker-compose.yml hono
 ~~~
+
+## View metrics
+
+Open [http://localhost:3000/dashboard/db/hono?orgId=1]() in a browser. Login is admin/admin.
```
Comment (sophokles73) [inline site/content/getting-started.md]: You should put the URL between the rounded brackets and put a descriptive text between the square brackets. We usually format literals monospaced, i.e. `admin/admin`

File: metrics/pom.xml [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <configuration>
+              <verbose>true</verbose>
+              <images>
+                <image>
+                  <alias>grafana</alias>
+                  <name>eclipsehono/grafana:${grafana.version}</name>
```
Comment (sophokles73) [inline metrics/pom.xml]: this should use `${project.version}` instead in order to correlate this image with the rest of the images we build.

**Comment by pellmann** [comment]
I have changed the dashboard - better widget with use of dropwizard meter. URL to dashboard int site text is changed, also.

**Pull request merged**